src/tye_lab_to_nwb/ast_ophys/tools/add_photon_series_to_processing.py

```diff
@@ -0,0 +1,44 @@
+from copy import deepcopy
+from typing import Optional
+
+import numpy as np
+from hdmf.backends.hdf5 import H5DataIO
+from neuroconv.tools import get_module
+from neuroconv.tools.roiextractors import add_imaging_plane
+from neuroconv.tools.roiextractors.imagingextractordatachunkiterator import ImagingExtractorDataChunkIterator
+from pynwb import NWBFile
+from pynwb.ophys import OnePhotonSeries
+from roiextractors import ImagingExtractor
+
+from neuroconv.utils import calculate_regular_series_rate
+
+
+def add_processed_one_photon_series(
```

Comment:
Remind me why this is necessary? Because we only dump `OnePhotonSeries` to `acquisition` in NeuroConv? And we don't have time left with the project to add that capability to NeuroConv's tools?

Reply:
Yes, exactly. As for future reference I can at least open an issue to add the option to write as One/TwoPhotonSeries as processing?

Reply:
Yes please, so we don't forget

Reply:
Issue created https://github.com/catalystneuro/neuroconv/issues/581